### PR TITLE
Adding SyncIQ Peer Certificate resource

### PR DESCRIPTION
**Pull request by rounak-adhikary**
image

Also removing tfplugindocs from go.mod

### CODE REVIEW

**Comment by rounak-adhikary** [comment]
run e2e test SyncIQCertificateResource

**Comment by rounak-adhikary** [comment]
run e2e test SyncIQCertificateResource

**Comment by rounak-adhikary** [comment]
run e2e test SyncIQCertificateResource

File: powerscale/provider/synciq_certificate_resource.go [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+	}
+
+	resp.State.RemoveResource(ctx)
+}
+
+// ImportState imports the resource.
+func (r *SyncIQPeerCertificateResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	state := models.SyncIQPeerCertificateResource{
+		ID:          types.StringValue(req.ID),
+		Path:        "/dummy",
```
Comment (shenda1) [inline powerscale/provider/synciq_certificate_resource.go:250]: Why these details are hardcoded?

Comment (rounak-adhikary) [reply powerscale/provider/synciq_certificate_resource.go:250]: Path is only required during create. The array does not return it on subsequent reads. So writing dummy string to state.
As for name and description, they get updated when terraform refresh happens as part of post-import action.

Comment (rounak-adhikary) [reply powerscale/provider/synciq_certificate_resource.go:250]: https://github.com/dell/terraform-provider-powerscale/pull/193/files#diff-b423b98401b6e6882a8db70fa093083eebcf80eafb50b68dd892c0f054af83b9R99-R112 tests that name and description gets updated

**Review by shenda1 — APPROVED** [review]

**Comment by rounak-adhikary** [comment]
run e2e pass

**Review by Krunal-Thakkar — APPROVED** [review]

File: examples/resources/powerscale_synciq_certificate/resource.tf [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+*/
+
+
+# Available actions: Create, Read, Update, Delete and Import.
+
+# Step 1: Upload a certificate to PowerScale.
+# This can be done using the file provisioner.
+resource "terraform_data" "cert" {
+  provisioner "file" {
+    source      = "/root/certs/pscale.crt" # certificate that user has been given
```
Comment (gokul-srivathsan) [inline examples/resources/powerscale_synciq_certificate/resource.tf:25]: Existing Comment: Certificate that user has been given
Proposed Comment: Certificate that user has given

**Pull request merged**